main.go

```diff
@@ -6,19 +6,21 @@ import (
 	"github.com/Sirupsen/logrus"
 	"github.com/rancher/agent/events"
 	"github.com/rancher/agent/register"
+	"github.com/rancher/agent/utilities/config"
 	"os"
 	"runtime"
 )
 
 var (
-	VERSION = "dev"
+	VERSION           = "dev"
```

Comment:
Why did you move these out of the method? It makes them "global" variables, which I prefer to avoid if possible. If you intentionally made the global so that they could be accessed, can you point to that spot in the code?